src/Npgsql/Internal/Converters/Temporal/PgTimestamp.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+
+namespace Npgsql.Internal.Converters;
+
+static class PgTimestamp
```

Comment:
Looks like we have a mostly-duplicate PgDateTime under NpgsqlTypes, used only by replication... Maybe deduplicate or move that one under replication, at least

Reply:
I'll move it to replication, it's mostly duplicate yeah (with the exception it directly maps to DateTime).